R/module_teal_data.R

```diff
@@ -125,11 +128,11 @@ srv_validate_reactive_teal_data <- function(id, # nolint: object_length
       }
 
       # to handle errors and qenv.error(s)
```

Comment:
Should we now handle `errors` separately? Or at least remove this part from the comment.

Reply:
There should not be a `shiny.error` at this stage as it is caught and converted into an empty `teal_data`